src/ReportManager/DetailsSection/index.js

```diff
@@ -249,7 +254,7 @@ const DetailsSection = ({
           renderSubmitButton={renderSchemaFormSubmitButton} />
     }
 
-    {!formSchema && !reportForm.is_collection && loadingSchema && <ResizeSpinLoader
+    {!formSchema && !reportForm.is_collection && !isNewFormSchemaSupportEnabled && loadingSchema && <ResizeSpinLoader
```

Comment:
I guess even if we use the new version of the schema we will want a loader while we load it right?